Maximum Pairwise Product In An Array #5871

Himanshi2997 opened the pull request:
<!--
Thank you for sending the PR! We appreciate you spending the time to work on these changes.

Help us understand your motivation by explaining why you decided to make this change.
You can learn more about contributing to NeoAlgo here: https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md

Happy Contributing!

-->

### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

yes

### Description

Implement a program in C++ to find the maximum product of two elements of an input array. The approach will be both time and space efficient. This proposal is for NeoAlgo/C-plus-plus.

### Checklist

- [ ] I've read the contribution guidelines.
- [ ] I've checked the issue list before deciding what to submit.
- [ ] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

#5871

Himanshi2997 committed:
Create maximum_pairwise_product file

Himanshi2997 committed:
updated README.md

Himanshi2997 committed:
Updated maximum_pairwise_product file

Himanshi2997 committed:
Merge branch 'master' into himanshi2

ankitaggarwal23 reviewed (commented):
why you are updating so many lines on readme?

Himanshi2997 commented:
Sir there was a conflict on readme file, to resolve it I deleted the
content between ==== and >>>>>master and just added my own file address.
Or else how do I resolve this?

On Sat, 1 May 2021, 03:03 Ankit Aggarwal, ***@***.***> wrote:

> Closed #6686 <https://github.com/TesseractCoding/NeoAlgo/pull/6686>.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/TesseractCoding/NeoAlgo/pull/6686#event-4672129734>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AQBH47VGPYH65JU54VPMY6LTLMO3DANCNFSM435MQVMQ>
> .
>

Himanshi2997 commented:
Sir please reply how to resolve conflict in readme file... The issue will
close soon.

On Sat, 1 May 2021, 03:16 Himanshi Rohatgi, ***@***.***>
wrote:

> Sir there was a conflict on readme file, to resolve it I deleted the
> content between ==== and >>>>>master and just added my own file address.
> Or else how do I resolve this?
>
> On Sat, 1 May 2021, 03:03 Ankit Aggarwal, ***@***.***>
> wrote:
>
>> Closed #6686 <https://github.com/TesseractCoding/NeoAlgo/pull/6686>.
>>
>> —
>> You are receiving this because you authored the thread.
>> Reply to this email directly, view it on GitHub
>> <https://github.com/TesseractCoding/NeoAlgo/pull/6686#event-4672129734>,
>> or unsubscribe
>> <https://github.com/notifications/unsubscribe-auth/AQBH47VGPYH65JU54VPMY6LTLMO3DANCNFSM435MQVMQ>
>> .
>>
>
